### PR TITLE
serial-fix

### PR DESCRIPTION
**Pull request by jakobrosenberg**
Couldn't push my branch upstream, so I'm using my personal account for now. (repo is private)

I can confirm that the fix works on Win x64 VS Code 1.59.1, but additional builds may be required for other archs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -169,7 +169,7 @@
     "postinstall": "pwsh -nop -f ./scripts/mp-download.ps1 -copyonly",
     "download-native": "pwsh -nop -f ./scripts/mp-download.ps1 -ElectronVersions 11.2.1 -IgnoreNodeVersion",
     "build": "babel src --out-dir lib --source-maps",
-    "rebuild": "electron-rebuild",
+    "install": "electron-rebuild -v 13.1.7",
```
Comment (elmasria) [inline package.json:172]: @jakobrosenberg I think it better to keep it `rebuild` to not force the end-user to use electron-rebuild on his end

**Review by elmasria — COMMENTED** [review]
please confirm when you are able to connect your device

**Comment by elmasria** [comment]
@jakobrosenberg  maybe you can merge to (https://github.com/pycom/pymakr-vsc/pull/148) directly?!

**Comment by jakobrosenberg** [comment]
Sure np!

**Pull request closed without merging**